xdp-forward/xdp-forward.c

```diff
@@ -217,6 +222,34 @@ static int do_load(const void *cfg, __unused const char *pin_root_path)
 		pr_info("Loaded on interface %s\n", iface->ifname);
 	}
 
+	/* Attach xdp_check_return to xdp_frame_return tracepoint and
+	 * pin the link to pin_root_path/xdp_check_return.
+	 */
+
+	return_prog = bpf_object__find_program_by_name(skel->obj,"xdp_check_return");
```

Comment:
The program is available in the skeleton as skel->progs.xdp_check_return - no need to use `bpf_object__find_program_by_name()`